packages/account/src/Sections/Security/ClosingAccount/__tests__/closing-account-has-pending-conditions.spec.tsx

```diff
@@ -0,0 +1,253 @@
+    getCFDAccountDisplay: jest.fn(() => 'Financial Demo'),
+}));
+
+describe('<ClosingAccountHasPendingConditions />', () => {
+    let store = mockStore({});
+    const mock_props = {
+        details: {
+            balance: {
+                CR123: {
+                    balance: 5000.0,
+                    currency: 'USD',
+                },
+                CR234: {
+                    balance: 0.18833116,
+                    currency: 'BTC',
+                },
+            },
+        },
+        onBackClick: jest.fn(),
+    };
+    store = mockStore({
+        client: {
+            mt5_login_list: [
+                {
+                    account_type: 'real',
+                    balance: 2,
+                    country: 'id',
+                    currency: 'USD',
+                    display_balance: '2.00',
+                    email: 'user@deriv.com',
+                    group: 'real\\p01_ts01\\financial\\svg_std-hr_usd',
+                    landing_company_short: 'svg',
+                    leverage: 1000,
+                    login: 'MTR456',
+                    market_type: 'financial',
+                    name: 'QA script userizbta',
+                    server: 'p01_ts01',
+                    server_info: {
+                        environment: 'Deriv-Server',
+                        geolocation: {
+                            group: 'all',
+                            location: 'Ireland',
+                            region: 'Europe',
+                            sequence: 1,
+                        },
+                        id: 'p01_ts01',
+                    },
+                    status: null,
+                    sub_account_category: '',
+                    sub_account_type: 'financial',
+                },
+            ],
+            dxtrade_accounts_list: [
+                {
+                    account_id: 'DXR345',
+                    account_type: 'real',
+                    balance: 2221,
+                    currency: 'USD',
+                    display_balance: '2221.00',
+                    landing_company_short: 'svg',
+                    login: '345',
+                    market_type: 'all',
+                },
+            ],
+            account_list: [
+                {
+                    loginid: 'CR123',
+                    is_virtual: 0,
+                    icon: 'usd',
+                    title: 'USD',
+                },
+                {
+                    loginid: 'CR234',
+                    is_virtual: 0,
+                    icon: 'btc',
+                    title: 'BTC',
+                },
+                {
+                    loginid: 'VRTC90000148',
+                    is_virtual: 1,
+                    icon: 'real',
+                    title: 'Real',
+                },
+            ],
+
+            is_eu: false,
+        },
+    });
+
+    it('should render the ClosingAccountHasPendingConditions component', () => {
+        const { container } = render(
```

Comment:
I assume we don't rely on container instead we use screen to use with expect. Right?